src/heads-up/components/properties-item.vue

```diff
@@ -30,15 +52,24 @@
               v-else
               class="properties-item__error-list"
             >
-              <!-- eslint-disable-next-line vue/no-v-html, vue/max-attributes-per-line -->
-              <li v-for="error in errors" :key="error.message" v-html="error.message" />
+              <li
+                v-for="error in errors"
+                :key="error.message"
+                v-html="error.message"
+              />
             </ul>
           </span>
+          <span
+            v-if="!errors && !warnings"
```

Comment:
`v-else`? Btw, als je zowel een warning als een error hebt, dan laat je momenteel alleen de error zien, is dit de bedoeling?